docs/System_Requirements.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: default
+datatable: true
+title: System requirements
+nav_order: 1
+keywords: system requirements
+last_updated: March 28, 2019
+summary: "."
+---
+# System requirements
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+## Matlab version
+
+MASH-FRET has been developed under Matlab Version 2012a, 2016a and is currently maintained under 2018b.
+
+**Note:** *MASH-FRET is currently tested to run under Matlab version R2016a and above. Tests of older versions of Matlab will follow asap.*
+
+## Matlab toolbox requirements for MASH-FRET
+
+Upon installation of Matlab the following toolbox need to be installed:
+
+<script>
+$(document).ready(function(){
+
+    $('table.display').DataTable( {
+        paging: true,
+        stateSave: true,
+        searching: true
+    }
+        );
+});
+</script>
+
+{% comment %} this is commented out
+
+<table id="sampleTable" class="display">
+   <thead>
+      <tr>
+         <th>Parameter</th>
+         <th>Description</th>
+         <th>Type</th>
+         <th>Default Value</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td>Parameter 1</td>
+         <td>Sample description
+         </td>
+         <td>Sample type</td>
+         <td>Sample default value</td>
+      </tr>
+      <tr>
+         <td>Parameter 2</td>
+         <td>Sample description
+         </td>
+         <td>Sample type</td>
+         <td>Sample default value</td>
+      </tr>
+    <tr>
+       <td>Parameter 3</td>
+       <td>Sample description
+       </td>
+       <td>Sample type</td>
+       <td>Sample default value</td>
+    </tr>
+      <tr>
+         <td>Parameter 4</td>
+         <td>Sample description
+         </td>
+         <td>Sample type</td>
+         <td>Sample default value</td>
+      </tr>
+   </tbody>
+</table>
+{% endcomment %}
+
+<div class="datatable-begin"></div>
+
+Toolbox | Matlab Version | MASH-FRET Version
```

Comment:
The table looks funny online, I think there is a problem in the layout.
Beside, some toolboxes with the same versions appearing several time in the table, maybe once is enough.
Otherwise, everything else looks fine to me, and necessary to have!